client/my-sites/backup/backup-contents-page/file-browser/use-prepare-download.ts

```diff
@@ -40,7 +40,11 @@ export const usePrepareDownload = ( siteId: number ) => {
 		dispatch( onPreparingDownloadError() );
```

Comment:
I can see part of `handleError` seems like an implementation detail of this hook but dispatching an error notice should best be done on the consumer side of the query hook. Similar to my comment above.

Reply:
Interesting. I just addressed this by receiving a custom error handling function as an argument on usePrepareDownload, so we could dispatch the notices on the consumer component as well. Thanks for the review!